oio/cli/object/object.py

```diff
@@ -31,6 +31,10 @@ class ContainerCommandMixin(object):
     def flatns_manager(self):
         return self.app.client_manager.flatns_manager
 
+    @property
+    def flatns_manager_override(self):
+        return self.app.client_manager.flatns_config
```

Comment:
Is that property necessary? Can we just call `self.app.client_manager.flatns_set_bits(bitlength)` or `self.app.client_manager.flatns_set_option(bits=bitlength)` a few lines below?